newIDE/app/src/LayersList/index.js

```diff
@@ -182,6 +215,13 @@ export default class LayersList extends Component<Props, State> {
           </FullSizeMeasurer>
           <Column>
             <Line justifyContent="flex-end" expand>
```

Comment:
I think I added a comment about having these buttons on two lines previously? Check if you can do that :)

Reply:
@4ian do you mean the button should be below the `Add a layer` button and not beside it?

Reply:
Yes because I'm concerned about small screens. Check how it looks if you resize the layers editor width to be very small?